### ISSUE TITLE
Disabled button text color?

### ISSUE THREAD

**Issue by butaminas** [nativescript-community/ui-material-components#186]
### Which platform(s) does your issue occur on?
Both
iOS 14 / Android 10
Both. iPhone 7+, iPhone 11 Pro Max, Xiaomi Redmi Note 8
### Please, provide the following version numbers that your issue occurs with:
CLI: 7.0.9-rc.0
Cross-platform modules: 7.0.3
Runtime(s): 7.0.0
Plugin(s):
```
  "dependencies": {
    "@mdi/font": "^5.6.55",
    "@nativescript-community/ui-material-activityindicator": "^5.0.6",
    "@nativescript-community/ui-material-bottomnavigationbar": "^5.0.6",
    "@nativescript-community/ui-material-bottomsheet": "^5.0.6",
    "@nativescript-community/ui-material-button": "^5.0.6",
    "@nativescript-community/ui-material-cardview": "^5.0.6",
    "@nativescript-community/ui-material-core": "^5.0.6",
    "@nativescript-community/ui-material-floatingactionbutton": "^5.0.6",
    "@nativescript-community/ui-material-progress": "^5.0.6",
    "@nativescript-community/ui-material-ripple": "^5.0.6",
    "@nativescript-community/ui-material-slider": "^5.0.6",
    "@nativescript-community/ui-material-tabs": "^5.0.6",
    "@nativescript-community/ui-material-textfield": "^5.0.6",
    "@nativescript-community/ui-material-textview": "^5.0.6",
    "@nativescript/core": "latest",
    "@nativescript/firebase": "11.0.0",
    "@nativescript/iqkeyboardmanager": "^2.0.0",
    "@nativescript/theme": "rc",
    "nativescript": "rc",
    "nativescript-vue": "latest",
    "nativescript-vue-navigator": "^1.2.0",
    "vuex": "^3.5.1"
  },
  "devDependencies": {
    "@babel/core": "~7.11.6",
    "@babel/preset-env": "~7.11.5",
    "@nativescript/android": "7.0.0",
    "@nativescript/ios": "7.0.0",
    "@nativescript/webpack": "~3.0.0",
    "babel-loader": "~8.1.0",
    "nativescript-vue-template-compiler": "~2.8.1",
    "node-sass": "^4.7.1",
    "scss-symbols-parser": "^2.0.1",
    "string-replace-loader": "^2.3.0",
    "vue-loader": "~15.9.3",
    "vue-property-decorator": "^9.0.0",
    "typescript": "^3.9.7",
    "@nativescript/types": "latest"
  }

```

Can't change MDButton text color neither using attributes (like `color="#fff"` nor CSS like `MDButton { color: #fff }`.

**Comment by farfromrefug** [comment]
@butaminas please can you share a example, possible sample app?

**Comment by butaminas** [comment]
@farfromrefug my bad, I think it was some sort rendering issue on my simulator where the color was not changing, the text color on button is actually working properly. However, it is not working when I am using `isEnabled` attribute to disable the button.

If I use `isEnabled` then the text color is dark / black and I can't change it  
Also, `:disabled` css rule didn't help.

My code

page.vue:

```
<MDButton height="50"
          flexGrow="0"
          width="125"
          borderRadius="25"
          text="Next"
          :isEnabled="!disabled"
          @tap="$navigator.navigate('/home', { clearHistory: true,
             transition: { name: 'flipLeft', duration: 500 }})"
/>
```

page.scss:

```

MDButton {
    color: $primary-dark;
    background-color: $primary;
    &:disabled {
      color: #fff;
    }
}
```

Sorry for the confusion!

**Comment by farfromrefug** [comment]
@butaminas talking iOS or Android

**Comment by butaminas** [comment]
@farfromrefug double checked to make sure - it is only not working on iOS, you can change the text color via **:disabled** css attribute.  
  
However, on Android, disabled button background color is light and you can't change it via `background-color` css (don't know if this is because how Android operate disabled buttons).

**Comment by farfromrefug** [comment]
@butaminas i see why! i need to think at about this. Will let you know

**Comment by farfromrefug** [comment]
@butaminas just tested it will work with `color`. You can set it in disabled. I added an example in the demo-vue app.
Now it did not work with backgroundColor or strokeColor. 
Made it work for backgroundColor. But cant do it for strokeColor for now.
Released 5.0.9
